core/domain/classifier_domain.py

```diff
@@ -277,6 +277,27 @@ def status(self):
     def training_data(self):
         return self._training_data
 
+    def update_state_name(self, state_name):
```

Comment:
@pranavsid98 actually I was thinking if we should do this or not. It is ok to keep this function around, but, we need to think how/if we are going to update the status of training_jobs linked with a particular state.